server/integration_tests/standalone/README.md

```diff
@@ -76,3 +76,17 @@ Note that leaving the `dc` flag to be empty will also default to the `--dc=main`
 ```shell
 python3 adversarial.py --mode=run_all --dc=sdg
 ```
+
```

Comment:
I guess no need to add the documentation below now?

Reply:
might be good to still describe what the test does? wdyt

Reply:
Sure, but can mention it running in gke and remove from line 87?

Reply:
done